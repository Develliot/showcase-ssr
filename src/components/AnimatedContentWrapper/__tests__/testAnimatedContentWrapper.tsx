import React from 'react';
import { shallow, ShallowWrapper } from 'enzyme';
import { AnimatedContentWrapper } from '../';

it('renders correctly with children', () => {
    const wrapper: ShallowWrapper = shallow(
        <AnimatedContentWrapper>
            <div>Hello</div>
            <div>World</div>
        </AnimatedContentWrapper>
    );
    expect(wrapper).toMatchSnapshot();
});
