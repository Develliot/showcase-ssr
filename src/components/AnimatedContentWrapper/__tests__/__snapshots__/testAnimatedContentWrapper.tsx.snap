// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly with children 1`] = `
<styles__ContentWrapperStyled
  visible={true}
>
  <div>
    Hello
  </div>
  <div>
    World
  </div>
</styles__ContentWrapperStyled>
`;
